### PR TITLE
Improve EndNote importer

### PR DESCRIPTION
**Pull request by Siedlerchr**
Fixes https://discourse.jabref.org/t/importing-endnote-label-field-to-jabref-from-xml-file/2734
Import label, publisher and import correct entry type for report
Fix issue with title having multiple formattings. 

I had to adjust the xsd because by default maxOccurs is assumed to be "1" 

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [x] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Failing unit tests are fromt he help pages #7759

File: src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (calixtus) [inline src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml:1]: Is the filename `EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml` intended? (same issue with corresponding `.bib`)

**Review by calixtus — COMMENTED** [review]
Looks good as far as I can see, just a question and mind the reviewdog (tabs instead of whitespaces).

File: src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml (hunk 1 above)
Comment (Siedlerchr) [reply src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestLabelAndMultiTitle Kopie.xml]: Thanks for spotting. Overlooked that

**Review by calixtus — APPROVED** [review]

**Comment by DominikVoigt** [comment]
LGTM!

**Review by DominikVoigt — APPROVED** [review]
Still LGTM!

**Pull request merged**